modules/fargate/monitoring/lambda_services_dashboard/templates/dashboard.json.j2

```diff
@@ -0,0 +1,149 @@
+{
+    "widgets": [
+    {% for item in template_data_list %}
```

Comment:
Do you think this dashboard could also be used with a regular ECS cluster, and should be a submodule under `modules/dashboard` for instance?